lib/internal/timers.js

```diff
@@ -289,31 +291,47 @@ function decRefCount() {
 // Schedule or re-schedule a timer.
 // The item must have been enroll()'d first.
 function active(item) {
-  insert(item, true, getLibuvNow());
+  schedule(item, true);
 }
 
 // Internal APIs that need timeouts should use `unrefActive()` instead of
 // `active()` so that they do not unnecessarily keep the process open.
 function unrefActive(item) {
-  insert(item, false, getLibuvNow());
+  schedule(item, false);
 }
 
 // The underlying logic for scheduling or re-scheduling a timer.
 //
 // Appends a timer onto the end of an existing timers list, or creates a new
 // list if one does not already exist for the specified timeout duration.
-function insert(item, refed, start) {
-  let msecs = item._idleTimeout;
+function schedule(item, refed, start) {
```

Comment:
IMO we should call this `insertGuarded`.

... and then possibly rename `insert` to `schedule` for both. (May cause too many backport conflicts?)